src/io/flutter/FlutterInitializer.java

```diff
@@ -130,26 +132,33 @@ public void runActivity(@NotNull Project project) {
     FlutterViewFactory.init(project);
 
     // If the project declares a Flutter dependency, do some extra initialization.
-    final PubRoot root = PubRoot.singleForProjectWithRefresh(project);
-    if (root != null && root.declaresFlutter()) {
-      // Set Android SDK.
-      if (root.hasAndroidModule(project)) {
-        ensureAndroidSdk(project);
-      }
+    for (Module module : ModuleManager.getInstance(project).getModules()) {
```

Comment:
re: better supporting multiple modules - this is fantastic! likely a cause of a long tail of issues for users

Reply:
Should we deprecate `PubRoot.singleForProjectWithRefresh`, or are there still valid use cases for it?

Reply:
Not sure!  Will think about it...